examples/app_with_observability/litellm_integration.py

```diff
@@ -17,7 +17,7 @@
 ]
 
 ag.init()
-litellm.callbacks = [ag.agenta_instrument_handler]
+litellm.callbacks = [ag.sdk.callbacks.litellm_handler()]
```

Comment:
Why not use a simpler syntax like `ag.callbacks.litellm_handler()` instead of the current `.sdk` structure with imports?

Reply:
agree